extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview.directive.ts

```diff
@@ -82,6 +90,14 @@ angular.module('oppia').directive('oppiaNoninteractiveSkillreview', [
               }
             });
           };
+          // Function to check whether the RTE modal or the concept card
+          // modal should be triggered.
+          ctrl.toggleModal = function(event) {
```

Comment:
This function name sounds like it will toggle a modal but actually, it's just doing a simple check, right? Should we rename it to `canOpenConceptCardModal`?

Reply:
Why not? I'll update the name.

Reply:
Fixed this.